### ISSUE TITLE
No Z progress after canceling NOMESH

### ISSUE THREAD

**Issue by romses** [paukstelis/Octoprint-Cancelobject#74]
I had a print with a failing support structure.
I decided to better have an ugly print, than a completely failed print.
So, I cancelled the support structure.
However, this prevented the Z-axis from being increased after each layer, and from then on all subsequent layers were printed at the same Z-height.

**Comment by paukstelis** [comment]
Yeah, I don't think there is anyway around that, since z-height changes are done in NONMESH blocks of gcode.
